pkg/reconciler/common/transformers.go

```diff
@@ -18,12 +18,36 @@ package common
 
 import (
 	"context"
-	mf "github.com/manifestival/manifestival"
+	"fmt"
+	"log"
+	"os"
+	"strings"
 
+	mf "github.com/manifestival/manifestival"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 	"knative.dev/pkg/logging"
 )
 
+const (
+	Retain int = iota
+	Overwrite
+
+	PipelinesImagePrefix = "IMAGE_PIPELINES_"
+	TriggersImagePrefix  = "IMAGE_TRIGGERS_"
+	AddonsImagePrefix    = "IMAGE_ADDONS_"
+
+	ArgPrefix   = "arg_"
+	ParamPrefix = "param_"
+
+	LabelProviderType     = "operator.tekton.dev/provider-type"
+	ProviderTypeCommunity = "community"
+	ProviderTypeRedHat    = "redhat"
+)
```

Comment:
I think we shouldn't use redhat or community terms here. Prolly it shoudl come from some flag or configmap.

Reply:
Or maybe we can include this logic in OpenShift.

Reply:
Looking at where it is used, I think we can define this in `openshift/*` rihgt ?

Reply:
Same for the other const.

Reply:
Updated PR